### ISSUE TITLE
Some programs need to be updated to make them platform independent

### ISSUE THREAD

**Issue by InfiniteCoder** [OpenGenus/cosmos#2084]
**This is a(n):**

- [ ] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [X] Proposal to the Repository

**Details:**

Some programs are using legacy and platform dependent code, which make them less helpful for people who do not use those platforms. For example, [Programs using conio.h](https://github.com/OpenGenus/cosmos/search?utf8=%E2%9C%93&q=conio.h&type=). These need to be updated

**Comment by abdouskamel** [comment]
I think that in the case of conio, it's just about showing the results of calcutation or getting input from the user, it doesn't have a real impact on algorithms.

**Comment by InfiniteCoder** [comment]
@abdouskamel It is, but it makes the program unusable (at least out of the box) on most of the systems. Turbo C is an outdated compiler which should not be used anymore. So code should be written in a platform independent way, so that anyone who uses any of the recent compilers and systems, like gcc, should be able to run the program.

**Comment by abdouskamel** [comment]
Of course, you cannot copy/paste those programs and run them. I saw that most programs include conio.h just to get access to getch(). This is easy to fix.

**Comment by jwdonahue** [comment]
I've cloned the repo and I have a lot of experience taking old compiler/OS specific code and making it portable.  Any priorities or should just search for conio.h and wade in?

**Comment by jwdonahue** [comment]
Oh, never mind, I just read the coding style guidelines.  No way I am ever going to write C or C++ code in KNF, it's arbitrary and indefensible.  No coding guidelines should ever require anything that common code beautifiers can't convert to/from.  Provide me with a tool that takes well written C/C++ code, and outputs acceptable KNF, and we can talk.
